src/app/item-page/item-page.resolver.ts

```diff
@@ -43,7 +44,13 @@ export class ItemPageResolver extends ItemResolver {
             this.router.navigateByUrl(itemRoute + subRoute);
           }
         }
-        return rd;
+
+        const copiedRd = _.cloneDeep(rd);
```

Comment:
@vidiecan I found used _cloneDeep_ in the DSpace7: https://github.com/DSpace/dspace-angular/blob/main/src/app/item-page/edit-item-page/item-metadata/item-metadata.component.ts#L129